AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -752,6 +752,9 @@ drimsim.com##.cookie[data-ix="cookie-close"]
 engie-energie.nl#$##cookieModal { display: none!important; }
 engie-energie.nl#$#.reveal-modal-bg { display: none!important; }
 engie-energie.nl#$#body { overflow: visible!important; }
+bauermedia.co.uk#$#.modal-backdrop { display: none !important; }
+bauermedia.co.uk#$#body { overflow: visible !important; }
+||bauermedia.co.uk/scripts/vendor.js
```

Comment:
you just break the site. Exclude required element for content blockers.

Reply:
What does it break? Everything works fine on my end.